app/Services/Csv/CsvData.php

```
@@ -0,0 +1,335 @@
+    {
+        return $this->headers[$index] ?? null;
+    }
+
+    /**
+     * Get header index by name
+     */
+    public function getHeaderIndex(string $headerName): ?int
+    {
+        $index = array_search($headerName, $this->headers, true);
+
+        return $index !== false ? $index : null;
```

PHPStan (2.1.15)

Method App\Services\Csv\CsvData::getHeaderIndex() should return int|null but returns int|string|null.


(return.type)